### ISSUE TITLE
Does not meet WCAG Standards for accessibility

### ISSUE THREAD

**Issue by Eng-Omar-Hussein** [jenkins-infra/stories#365]
### Reproduction steps

**The Color Contrast accessibility issue for the "View All Stories" and "Explore Map" buttons**

<img width="1189" height="516" alt="Image" src="https://github.com/user-attachments/assets/ce2488e1-e64f-4caf-94f9-bae0dea0785d" />

### Expected Results

Text elements should have a color contrast ratio adhering to WCAG 2.1 AAA success criteria.

### Actual Results

<img width="1682" height="742" alt="Image" src="https://github.com/user-attachments/assets/c9f03ef5-0e4c-4562-9886-d2507636de23" />

### Anything else?

Lighthouse flags this as an Accessibility issue, so Improving color contrast is essential for inclusive design

**Comment by Eng-Omar-Hussein** [comment]
Hi @krisstern, what about this issue!

**Comment by krisstern** [comment]
Hi @Eng-Omar-Hussein we could consider this issue to be valid. 

Please be aware that we are currently busy evaluating all submitted Jenkins GSoC applications for 2026 to come up with some rankings, so our bandwidth for handling issues/pull requests has been reduced.

**Comment by Eng-Omar-Hussein** [comment]
> Hi [@Eng-Omar-Hussein](https://github.com/Eng-Omar-Hussein) we could consider this issue to be valid.
> 
> Please be aware that we are currently busy evaluating all submitted Jenkins GSoC applications for 2026 to come up with some rankings, so our bandwidth for handling issues/pull requests has been reduced.

Sorry about that but I have applied for GSoC2026, so I aim to get my first merge/contribution to strengthen my application/proposal

**Comment by krisstern** [comment]
Actually as long as you have some git history of both opened or merged PRs that are of good quality that should be sufficient. This chasing does not necessarily help with your application, and may impact it negatively.
